app/src/main/java/com/poupa/vinylmusicplayer/provider/StaticPlaylist.java

```diff
@@ -34,7 +38,13 @@ private static List<StaticPlaylist> importDevicePlaylists(@NonNull final Context
         final SharedPreferences preferences = getPreferences();
         final boolean noMigrationMarker = !preferences.contains(PREF_MIGRATED_STATIC_PLAYLISTS);
 
-        Set<String> previouslyMigratedNames = new HashSet<>(preferences.getStringSet(PREF_MIGRATED_STATIC_PLAYLISTS, new HashSet<>()));
+        Gson gson = new Gson();
+
+        //Set<String> previouslyMigratedNames = new HashSet<>(preferences.getStringSet(PREF_MIGRATED_STATIC_PLAYLISTS, new HashSet<>()));
```

Comment:
Hmm, not sure what's the issue with StaticPlaylist and your intention here?

Reply:
I tought about, that this can cause incompatibilities. I think as your next comment mentions, we can avoid data type issues with json, so this change becomes unnecessary.
I just wanted to avoid fiddeling around with Sets if I want to put the String of the exported file back into the shared prefs.